etna/records/templatetags/records_tags.py

```diff
@@ -54,6 +54,31 @@ def as_label(record_field_name: str) -> str:
 
 
 @register.simple_tag
-def level_name(level_code: int) -> str:
+def level_name(level_code: int, is_tna: bool) -> str:
     """returns level as a human readable string"""
-    return LevelKeys["LEVEL_" + str(level_code)].value
+    if is_tna:
+        return LevelKeys["LEVEL_" + str(level_code)].value
+    else:
+        return NonTNALevelKeys["LEVEL_" + str(level_code)].value
+
+
+@register.simple_tag
+def breadcrumb_items(hierarchy: list, is_tna: bool, current_item: Record) -> list:
+    """Returns breadcrumb items depending on position in hierarchy
+    Update tna_breadcrumb_levels or oa_breadcrumb_levels to change the levels displayed
+    """
+    items = []
+    tna_breadcrumb_levels = [1, 2, 3]
+    oa_breadcrumb_levels = [1, 2, 5]
+    for record in hierarchy:
```

Comment:
```suggestion
    for hierarchy_record in hierarchy:
```

use `hierarchy_record`  to make it more clear in the loop.